exonum-java-proofs/src/test/java/com/exonum/binding/storage/serialization/StandardSerializersRoundtripTest.java

```diff
@@ -0,0 +1,56 @@
+package com.exonum.binding.storage.serialization;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+import com.exonum.binding.hash.HashCode;
+import com.google.common.collect.Lists;
+import java.util.List;
+import org.junit.Test;
+
+public class StandardSerializersRoundtripTest {
+
+  @Test
+  public void roundtripLongTest() {
+    List<Long> valuesToTest = Lists.newArrayList(
```

Comment:
ImmutableList.of() (newArrayList is mutable, but it does not seem to be intended)?